app/javascript/packs/application.js

```diff
@@ -7,9 +7,11 @@ import Rails from "@rails/ujs"
 
 Rails.start()
 
-import 'uswds/src/stylesheets/uswds.scss'
+import 'css/application.scss'
 import 'uswds/dist/img/icon-dot-gov.svg'
 import 'uswds/dist/img/us_flag_small.png'
 import 'uswds/dist/img/icon-https.svg'
 
-require('uswds')
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
oops, banner expansion wasn't actually working previously. Waiting until DOM is loaded before loading USWDS javascript fixed that.

Reply:
good catch!